test/jdk/sun/security/ssl/CipherSuite/NoDesRC4DesEdeCiphSuite.java

```diff
@@ -45,8 +45,6 @@
 
 public class NoDesRC4DesEdeCiphSuite {
```

Comment:
I suggest making a new abstract class with the general test methods, and then making this a subclass of it.

Reply:
Will do